### PR TITLE
Adds Treasure Ruby testnet

### PR DESCRIPTION
**Pull request by mohamed-mehany**
This PR adds support for Treasure Ruby testnet chain.

### Motivation
[SHIP-2409](https://smartcontract-it.atlassian.net/browse/SHIP-2409)

### CODE REVIEW

**Comment by KodeyThomas** [comment]
Just as a FYI treasure mainnet is currently not available so we're not including it in this PR

File: selectors.yml [hunk 1]
```diff
@@ -195,6 +195,9 @@ selectors:
   168587773:
     selector: 2027362563942762617
     name: "ethereum-testnet-sepolia-blast-1"
+  978657:
+    selector: 10443705513486043421
+    name: "ethereum-testnet-sepolia-arbitrum-1-treasure-ruby"
```
Comment (KodeyThomas) [inline selectors.yml:200]: going by the convention shouldn't this be `ethereum-testnet-ruby-arbitrum-1-treasure-1`?

Comment (mohamed-mehany) [reply selectors.yml:200]: I'm using the same RDD naming: https://github.com/smartcontractkit/reference-data-directory/tree/master/ethereum-testnet-sepolia-arbitrum-1-treasure-ruby/linkTokens

If we rename it here, we might need to rename it for RDD as well.

Comment (mateusz-sekara) [reply selectors.yml:200]: RDD promised to follow the chain-selectors naming convention, we've agreed chain-selectors to be the source of truth

Comment (mateusz-sekara) [reply selectors.yml:200]: @andrevmatos your thought on that?

Comment (mohamed-mehany) [reply selectors.yml:200]: @KodeyThomas Following the convention, it would be then `ethereum-testnet-sepolia-arbitrum-1-treasure-1` right?

**Review by KodeyThomas — APPROVED** [review]

File: selectors.yml (hunk 1 above)
Comment (mohamed-mehany) [reply selectors.yml]: Great, resolving this by changing the name to: `ethereum-testnet-sepolia-arbitrum-1-treasure-1`

**Review by mateusz-sekara — APPROVED** [review]

**Pull request merged**